synapse/cortex.py

```diff
@@ -3054,9 +3054,18 @@ async def _onload():
 
                 logextra = await self.getLogExtra(pkg=name, vers=pkgvers)
 
-                if inits is not None:
-                    varname = inits['key']
-                    curvers = await self.getStormPkgVar(name, varname, default=-1)
+                verskey = 'storage:version'
+
+                if inits is None:
+                    await self.setStormPkgVar(name, verskey, -1)
+
+                else:
+                    if (key := inits.get('key')) is not None:
+                        s_common.deprecated('storm package inits.key', eolv='3.0.0')
+                        if key != verskey and (valu := await self.popStormPkgVar(name, key)) is not None:
+                            await self.setStormPkgVar(name, verskey, valu)
+
+                    curvers = await self.getStormPkgVar(name, verskey, default=-1)
```

Comment:
I think this default needs to be something other than -1 now to identify new packages

Reply:
we can only go negative so i think -2 is the next option